src/components/views/userPublicProfile/ProjectsTable.tsx

```diff
@@ -66,7 +66,7 @@ const ProjectsTable: FC<IProjectsTable> = ({
 			title = 'Listed';
 		} else if (listed === null) {
 			color = semanticColors.blueSky;
-			title = 'Waiting for review';
+			title = 'Draft';
```

Comment:
What if a project is published but not listed yet? Do you show `Draft` for that?
Maybe you should add another condition to check if the project is `drafted` or not

Reply:
Added the draft case and maintained the Waiting for review tag.